components/system-broker/pkg/server/server.go

```diff
@@ -39,15 +40,18 @@ type Server struct {
 	shutdownTimeout time.Duration
 }
 
-func New(c *Config, service log.UUIDService, routesProvider ...func(router *mux.Router)) *Server {
+func New(c *Config, service log.UUIDService, forwardHeaders []string, routesProvider ...func(router *mux.Router)) *Server {
```

Comment:
We can make New have a slice of middlewares as argument insted of providing the forwardHeaders. This way that middleware (which is specific for the system broker) can be provided in the main and other brokers can add their own middlewares.